public/assets/base.css

```diff
@@ -1,16 +1,3 @@
-/*
-    * Web Fonts from colophon-foundry.org
-    *
-    * The fonts included in this stylesheet are subject to the End User License you purchased
-    * from Colophon Foundry. The fonts are protected under domestic and international trademark and
-    * copyright law. You are prohibited from modifying, reverse engineering, duplicating, or
-    * distributing this font software.
-    *
-    * (c) 2023 Colophon Foundry
-    *
-    * Licenced to Trustmachines,
-*/
-
 @font-face {
```

Comment:
Maybe we should move these `font` declarations somewhere also? Inside `theme/typography` perhaps?